### PR TITLE
Slack native poller connector

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes

  - add Slack native poller data connector

### CODE REVIEW

**Comment by vmanojreddy** [comment]
Please Provide Sample data.

**Comment by vu-socprime** [comment]
> Please Provide Sample data.

@vmanojreddy Done

File: Solutions/SlackAudit/Data Connectors/SlackNativePollerConnector/azuredeploy_Slack_native_poller_connector.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                            "query": "{{graphQueriesTableName}}\n| sort by TimeGenerated desc"
+                        }
+                    ],
+                    "dataTypes": [
+                        {
+                            "name": "{{graphQueriesTableName}}",
+                            "lastDataReceivedQuery": "{{graphQueriesTableName}}\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+                        }
+                    ],
+                    "connectivityCriterias": [
```
Comment (NikTripathi) [inline Solutions/SlackAudit/Data Connectors/SlackNativePollerConnector/azuredeploy_Slack_native_poller_connector.json:56]: Please remove the s from the end. Thank you.

**Review by NikTripathi — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
@oshezaf Please review Parser.

File: Solutions/SlackAudit/Data Connectors/SlackNativePollerConnector/azuredeploy_Slack_native_poller_connector.json (hunk 1 above)
Comment (vu-socprime) [reply Solutions/SlackAudit/Data Connectors/SlackNativePollerConnector/azuredeploy_Slack_native_poller_connector.json]: done

**Comment by oshezaf** [comment]
@NikTripathi : See my comment on https://github.com/Azure/Azure-Sentinel/pull/3504

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**